apps/dotcms-ui/src/app/view/components/dot-content-compare/components/fields/dot-content-compare-binary-field/dot-content-compare-binary-field.component.html

```diff
@@ -0,0 +1,8 @@
+<img [src]="fileURL" *ngIf='!imageError else fallback' (error)="handleError()"/>
+
+<ng-template #fallback>
+    <span [innerHTML]="label"></span>
+    <a [href]="fileURL" download>
+        <button pbutton class="p-button-secondary p-button p-component">Download</button>
+    </a>
```

Comment:
This is not valid HTML. You can't have a button inside an a, I think we have styles to make the a looks like a button.